pkg/volume/util/util.go

```diff
@@ -1023,3 +1026,18 @@ func MapBlockVolume(
 
 	return nil
 }
+
+func IsCRDInstalled(name, groupVersion string, client discovery.DiscoveryInterface) (bool, error) {
+	resources, err := client.ServerResourcesForGroupVersion(groupVersion)
```

Comment:
this doesn't work, this commit can be ignored pending discussion re: https://github.com/kubernetes/kubernetes/issues/69539#issuecomment-438472592